File: auth-service/src/main/java/com/sudo248/authservice/contronller/dto/AddressDto.java
package com.sudo248.authservice.contronller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AddressDto {
    private String addressId = "";

    private int provinceID = 0;

    private int districtID = 0;

    private int wardCode = 0;

    private String provinceName = "";

    private String districtName = "";

    private String wardName = "";

    private String address = "";

    private LocationDto location;

    private String fullAddress = "";
}

File: client/SocStaff/app/src/main/java/com/sudo248/soc_staff/data/dto/user/AddressDto.kt
package com.sudo248.soc_staff.data.dto.user

import com.sudo248.soc_staff.domain.entity.user.Location

data class AddressDto(
    val addressId: String = "",
    val provinceID: Int = 0,
    val districtID: Int = 0,
    val wardCode: String = "",
    val provinceName: String = "",
    val districtName: String = "",
    val wardName: String = "",
    val address: String = "",
    val location: Location = Location(),
    val fullAddress: String = "",
)